lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -18,13 +27,67 @@
                         courseID: this.courseID,
                         discussionTopicID: this.discussionTopicID,
                         canCreateComment: canPostToTeam,
-                        canCreateSubComment: canPostToTeam
+                        canCreateSubComment: canPostToTeam,
+                        country: this.model.get('country'),
+                        language: this.model.get('language'),
+                        membershipText: interpolate(
+                            // Translators: The following message displays the number of members on a team.
+                            ngettext(
+                                '%(member_count)s / %(max_member_count)s Member',
+                                '%(member_count)s / %(max_member_count)s Members',
+                                this.maxTeamSize
+                            ),
+                            {member_count: this.memberships.length, max_member_count: this.maxTeamSize}, true
```

Comment:
Can we make this a function and move it to a helper file? This code is in `TeamMembershipView` as well.